title: miRNA
date: 2015-12-01 14:06:06
tags: miRNA
categories: molecular biology
---
##简介
MicroRNAs (miRNAs) are short in sequence and are generated by enzymatic excision from precursor transcripts called primary miRNAs (pri-miRs), which until now had been assumed not to encode any proteins.
<a href="/nature/journal/v520/n7545/full/nature14346.html">Lauressergues et al</a> find that plant pri-miRNAs contain short open reading frame sequences that encode regulatory peptides. This is the first report of a functional peptide being encoded by a pri-miR and provides a fresh perspective on the significance of pri-miR regions beyond those that directly give rise to miRNAs.
![](http://7xk19o.com1.z0.glb.clouddn.com/miRNA.png)
图片注释：MicroRNAs and their associated peptides. The precursors of plant microRNAs (miRNAs) are
pri-miR sequences, which are transcribed from DNA by the enzyme RNA polymerase II (Pol II). They are
then modified by capping and addition of a poly(A) tail. The miRNA duplexes are subsequently excised by
the enzyme Dicer-like1 and transported to the cytoplasm; other parts of the pri-miR are degraded. After
further processing, the resulting miRNA sequence guides repression of gene expression as part of the
RISC complex. Lauressergues et al.report that some pri-miRs contain short open reading frame (ORF)
sequences that can produce peptides (miPEP). The miPEPs enhance expression of the pri-miR, leading
to more miRNA and so more effective cleavage of the target gene's messenger RNA. Such ORFs may avoid
degradation as part of pri-miRs that might exit the nucleus without being processed by Dicer-like1.
##关键信息总结
1）	Both plant and animal pri-miRs are transcribed from DNA in the nucleus by the enzyme RNA polymerase II.
2）	The structured (fold-back) region of the transcript surrounding the miRNA sequence is recognized and processed by one of two enzymes-Drosha or Dicer-like1(In plants, Dicer-like1 cuts out the miRNA in a duplex form).
3)	Transporter proteins export the excised sequences to the cytoplasm, where they are further processed before becoming competent to guide the RNA-induced silencing complex (RISC) in repressing target genes through either cleavage or translational repression of their mRNAs.
4)	The miPEPs had the same tissue distribution as their associated mature miRNAs and enhanced the expression and effectiveness of these miRNAs. Moreover, the miPEPs promoted the transcription of their corresponding pri-miR, rather than enhancing miRNA stability.
##miRNA合成途径
miRNA的合成大体经过：初级miRNA(pri-miRNA, primary miRNA)→前体miRNA(pre-miRNA, precursor-miRNA)→成熟miRNA(miRNA, mature miRNA)。
其具体合成途径存在两种形式：经典合成途径和Mirtron合成途径；
![](http://7xk19o.com1.z0.glb.clouddn.com/nihms311773f1.jpg)
###经典合成途径
来源：基因间隔区或编码基因内含子中；
细胞核内编码miRNA 的基因通过RNA聚合酶II 或RNA聚合酶III 转录生成初级miRNA(pri-miRNA)，pri-miRNA与来自蛋白质编码基因mRNA相似，有5' 端帽式结构和3' 端多聚腺苷酸尾结构，长度可达数千个碱基。接着pri-miRNA在一种RNaseIII(Drosha 酶)和它的伴侣分子(DiGeorge syndrome critical region gene 8,DGCR8)组成的复合物作用下，剪切为70~80 个核苷酸长度、具有茎环结构的miRNA 前体(pre-miRNA)。miRNA 前体在Ran-GTP 依赖的核质/ 细胞质转运蛋白——输出蛋白5(exportin 5)的作用下，从核内运输到胞质中。随后miRNA前体在另一种RNaseIII(Dicer酶)的作用下被剪切成21~25 个核苷酸长度，而且其5'端磷酸化和3'端有2 nt 的悬垂序列，类似于siRNA 的不完全配对的双链RNA，它们是由成熟miRNA与miRNA\* 组成的二聚体，miRNA\*是pre-miRNA中的一段RNA，其位置恰好与成熟的miRNA 相对。最后在RNA解旋酶作用下生成成熟miRNA 和miRNA\*，成熟miRNA结合到RNA诱导的基因沉默复合物(RNA-induced silencing complex, RISC)中发挥作用，miRNA\* 则被降解。
![](http://7xk19o.com1.z0.glb.clouddn.com/ruby-et-al-nature-mirtron.jpg)
###Mirtron途径
2007年，Ruby等在研究黑腹果蝇和秀丽隐杆线虫的小RNA 序列时首次发现mirtron，是一类定位于mRNA编码基因内含子内的miRNA，与**经典miRNA合成相比其合成过程不需要经过Drosha酶切割**，其形式功能与miRNA相同，具有以下特点：
(1)来自长度为56 nt 的内含子序列；
(2)能够形成miRNA:miRNA*复合物；
(3)内含子序列及其二级结构非常保守；
(4)内含子存在于基因组中的外显子中间并且有"GU-AG"的典型内含子特点；
(5)最后形成的成熟miRNA总来自于3' 端侧的序列；
(6)合成需要Ldbr(lariat-debranching enzyme,套索分支酶)(参与内含子剪接) 的作用；
(7)与经典的miRNA一样，依赖Ran-GTP/ 输出蛋白5 的转运机制转运出核，在Dicer 酶的作用下成为成熟的miRNA。
<code>同时，mirtron在哺乳动物中和在无脊椎动物中有明显的不同，如：</code>
(1)哺乳动物中的mirtron途径合成的功能miRNA大多数来自mirtron的5'茎序列；
(2)不管哺乳动物中的mirtron3' 茎序列能否成为成熟miRNA，它的起始核苷酸总是尿嘧啶(U)或胞嘧啶(C); 
(3)哺乳动物中mirtron悬垂序列(overhang, 发夹结构末端突出序列)大多数为单核苷酸(G)，只有少数是双核苷酸(AG)；
(4)哺乳动物中mirtron比经典的miRNA前体和无脊椎动物mirtron的GC 含量要高，自由能要低。
##阻断miRNA对mRNA的调控作用
###miRNA海绵(miRNA sponges)
miRNA海绵是一种miRNA靶基因的竞争性抑制剂，是将若干个miRNA的反义序列串联在一起，连接到合适的载体中表达，其转录物"吸附"相应的miRNA，与miRNA靶基因形成竞争，导致靶基因去抑制化。
<a href="http://d.wanfangdata.com.cn/Periodical/gwyx-fzsw201404010" target="_blank">miRNA海绵表达载体的构建策略</a>
###circRNA
环状RNA（circRNA）对miRNA的吸附作用。<i class="fa fa-link" aria-hidden="true"></i>http://tiramisutes.github.io/2016/04/12/Circular-RNAs/
##miRNA数据库
miocroRNA databases:<a target="_blank" href="http://micrornadatabases.com/Databases.html">http://micrornadatabases.com/Databases.html</a>
##miRNA相关工具
###mirtron prediction
<a href="http://cbio.mskcc.org/leslielab/mirtrons/">Mirtron'SVM'prediction</a>
###MicroRNA Target Prediction
<a href="http://www.microrna.org/microrna/home.do">miRanda</a> &#8212; miRNA target prediction for human, drosophila and zebrafish genomes
<a href="http://www.mirbase.org/">miRBase</a> &#8212; a comprehensive repository for miRNAs and their predicted targets
<a href="http://mirdb.org/miRDB/">miRDB</a> &#8212; an online database for miRNA target prediction and functional annotations in animals
<a href="http://mirnamap.mbc.nctu.edu.tw/">miRNAMap</a> &#8212; a genomic maps of microRNA genes and their target genes in mammalian genomes
<a href="http://watson.compbio.iupui.edu:8080/mir2disease/">miR2Disease</a> &#8212; a database providing comprehensive resource of miRNA deregulation in various human diseases
<a href="http://diana.cslab.ece.ntua.gr/tarbase/">TarBase</a> &#8212; a comprehensive database of experimentally supported animal microRNA targets
<a href="http://pictar.mdc-berlin.de/">PicTar</a> &#8212; microRNA targets for vertebrates, fly and nematodes 
<a href="http://www.targetscan.org/">TargetScan</a> &#8212; a search for the presence of conserved sites that match the seed of each miRNA
<a href="http://www.russell.embl.de/miRNAs/">Target Gene Prediction at EMBL</a> &#8212; miRNA-Target predictions for Drosophila miRNAs
###Databases for microRNA Expression
<a href="http://www.microrna.org/microrna/getExprForm.do">microRNA.org</a> &#8212; predicted microRNA targets &amp; target downregulation scores. Experimentally observed expression patterns
<a href="http://202.38.126.151/hmdd/mirna/md/">HMDD</a> &#8212; Human MicroRNA Disease Database (HMDD) is a database that contains the experimentally supported miRNA-disease
association data, which are manually curated from publications. The dysfunction evidence or miRNAs and literature PubMed ID are also given
<a href="http://202.38.126.151/hmdd/mirna/tf/">TransmiR</a> &#8212; a web query-driven database integrating the experimentally supported transcription factor and miRNA regulator relations
###RNA Secondary Structure Prediction
<a href="http://diana.pcbi.upenn.edu/cgi-bin/micro_t.cgi/">DIANA MicroTest</a> &#8212; a prediction of miRNA-mRNA interaction 
<a href="http://mfold.bioinfo.rpi.edu/">mfold</a> &#8212; tools for predicting the secondary structure of RNA and DNA, mainly by using thermodynamic methods
<a href="http://mirna.imbb.forth.gr/microinspector/">microInspector</a> &#8212; a web tool for detection of miRNA binding sites in an RNA sequence
<a href="http://www.mirnainfo.com/EnergyCalc.aspx">miRNA Bioinfor</a> &#8212; miRNA End Energy calculator which takes miRNA duplex to calculate free energy for 5 base pairs at
one end plus a dangling nucleotide
<a href="http://mirrim.ncrna.org/">miRRim</a> &#8212; a method for detecting miRNA foldbacks based on hidden Markov model (HMM)
<a href="http://mxscarna.ncrna.org/">MXSCARNA</a> &#8212; a multiple alignment tool for RNA sequences using progressive alignment based on pairwise structural alignment algorithm of SCARNA. Good for large scale analyses.
<a href="http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/">RNAhybrid</a>&#8212; a tool for finding the minimum free energy hybridisation of a long and a short RNA
###MicroRNA Homologous Prediction
<a href="http://groups.csail.mit.edu/pag/mirnaminer/">miRNAminer</a> &#8212; a web-based tool used for homologous miRNA gene search in several species
<a href="http://people.csail.mit.edu/akiezun/miRviewer/">miRviewer</a> &#8212; a global view of homologous miRNA genes in many species
<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2676257/pdf/1471-2105-10-105.pdf">RISCbinder</a>&#8212; prediction of guide strand of microRNAs
<a href="http://tagc.univ-mrs.fr/mireval/">Mireval</a> &#8212; Sequence evaluation of microRNA properties
###MicroRNA Deep Sequencing
<a href="http://web.bioinformatics.cicbiogune.es/microRNA/miRanalyser.php">miRanalyzer</a>&#8212; A microRNA detection and analysis tool for next-generation sequencing experiments 
<a href="http://ibis.tau.ac.il/miRNAkey/">miRNAkey</a>&#8212; A software pipeline for the analysis of microRNA Deep Sequencing data
<a href="http://www.mdc-berlin.de/en/research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/index.html">miRDeep</a>&#8212; Discovering known and novel miRNAs from deep sequencing data
##miRNA百科
<i class="fa fa-link" aria-hidden="true"></i><a href="http://www.gene-quantification.de/micro-rna.html" target="_blank">microRNA.gene-quantification.info</a>

参考文献：
<i class="fa fa-link" aria-hidden="true"></i><a href="http://www.nature.com/nature/journal/v520/n7545/full/nature14378.html" target="_blank">Plant biology: Coding in non-coding RNAs</a>
<i class="fa fa-link" aria-hidden="true"></i><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3185189/" target="_blank">Mirtrons: microRNA biogenesis via splicing</a>

