title: De Noveè½¬å½•ç»„ç»„è£…è´¨é‡è¯„ä¼°
Total word: WordCount
Read time: Min2Read
date: 2016-10-30 23:00:17
tags: De Nove
categories: Bioinformatics
---
æ— å‚De Noveç»„è£…é€šå¸¸ç”¨åˆ°**<a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki" target="_blank">Trinity</a>**è½¯ä»¶,ç»„è£…è¿‡ç¨‹ä¸­æœ€é‡è¦çš„ä¸¤ä¸ªå‚æ•°å°±æ˜¯
<span class="myCode">--min_kmer_cov</span>å’Œ<span class="myCode">--min_glue</span>ã€‚ä¸ºç»„è£…å‡ºé«˜è´¨é‡ç»“æœæˆ‘ä»¬é€šå¸¸éœ€è¦å»å°è¯•ç”¨ä¸åŒçš„å‚æ•°ï¼Œgithubä¸Šä¹Ÿæœ‰è½¯ä»¶å¼€å‘è€…è®¨è®ºå…³äºè¿™ä¸¤ä¸ªå‚æ•°ï¼š**<a href="https://github.com/trinityrnaseq/trinityrnaseq/issues/92" target="_blank">Optimizing parameters</a>**å¯ä¾›å‚è€ƒï¼Œå…¶å®é—®é¢˜æœ€ç»ˆä¹Ÿå°±å½’ç»“ä¸ºä½ æ˜¯å¦å…³å¿ƒä½ æ•°æ®ä¸­çš„ä½ä¸°åº¦è½¬å½•æœ¬ã€‚
æ­¤å¤–ä½œè€…ä¹Ÿæä¾›äº†ä¸€ç³»åˆ—æ–¹æ³•æ¥è¯„ä¼°ç»„è£…è´¨é‡**<a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki/Transcriptome-Assembly-Quality-Assessment" target="_blank">Transcriptome Assembly Quality Assessment</a>**ï¼Œæ€»å…±åˆ—å‡ºæœ‰6ç§æ–¹æ³•å¯å¯¹ä¸åŒå‚æ•°çš„ç»„è£…ç»“æœè¿›è¡Œè¯„ä¼°,çœ‹å®Œåç»¼åˆæ€»ç»“å‡ºå…¶ä¸­4ç§è¯„ä¼°æ–¹æ³•ã€‚
##Assessing the Read Content of the Transcriptome Assembly
``` bash
bowtie2-build  ../trinity_out_dir${i}/Trinity.fasta ../trinity_out_dir${i}/Trinity.fasta
bowtie2 --local --no-unal -p ${cpu} -x  ../trinity_out_dir${i}/Trinity.fasta -q -1 ${left} -2 ${right} \
     | samtools view -Sb - | samtools sort -no - - > bowtie2.nameSorted.bam
#å‚çœ‹proper pairs readsæ•°é‡å’Œç™¾åˆ†æ¯”
${TRINITY_DIR}/util/SAM_nameSorted_to_uniq_count_stats.pl  bowtie2.nameSorted.bam
grep "^proper_pairs" Read-Representation.out
```
**ç¬¬äºŒæ­¥çš„bowtie2æ¯”å¯¹åºåˆ—åˆ°ç»„è£…è½¬å½•æœ¬ç»“æœæ—¶å¯é€‰éƒ¨åˆ†æ•°æ®æ¥æ¯”å¯¹ï¼Œè¿™æ ·å¯å¤§å¤§é™ä½æ¯”å¯¹è€—æ—¶ã€‚**
##Full-length transcript analysis for model and non-model organisms using BLAST+
``` bash
blastall -p blastx -i ./trinity_out_dir${i}/Trinity.fasta  -d ${uniprot} -v 1 -b 1 -m 8 -e 1e-5 -a ${cpu} -F F -o uniprot_sprot.fasta_blastx.outfmt8
${TRINITY_DIR}/util/analyze_blastPlus_topHit_coverage.pl uniprot_sprot.fasta_blastx.outfmt8 ./trinity_out_dir${i}/Trinity.fasta /public/home/cotton/public_data/SwissProt/uniprot_sprot.fasta
${TRINITY_DIR}/util/misc/blast_outfmt6_group_segments.pl \
      ./uniprot_sprot.fasta_blastx.outfmt8 ./trinity_out_dir${i}/Trinity.fasta uniprot_sprot.fasta > ./uniprot_sprot.fasta_blastx.outfmt8.grouped
${TRINITY_DIR}/util/misc/blast_outfmt6_group_segments.tophit_coverage.pl ./uniprot_sprot.fasta_blastx.outfmt8.grouped
```
##Compute DETONATE scores
RSEM-EVALè½¯ä»¶å¯¹äºåŒç«¯readsæ•°æ®éœ€è¦æä¾›ä¸€ä¸ªaverage fragment lengthå€¼ï¼Œå¯å‚è€ƒæˆ‘çš„å¦ä¸€ç¯‡åšæ–‡**<a href="http://tiramisutes.github.io/2016/09/19/Insert-Size.html" target="_blank">è¯„ä¼°æ–‡åº“ Average Insert Size</a>**æ¥è®¡ç®—å¾—åˆ°æ­¤å€¼ã€‚
``` bash
rsem-eval-estimate-transcript-length-distribution ./trinity_out_dir${i}/Trinity.fasta ./RSEM-EVAL${i}/length_distribution_parameter.txt
rsem-eval/rsem-eval-calculate-score -p 1 \
              --transcript-length-parameters ./RSEM-EVAL${i}/length_distribution_parameter.txt \
              --paired-end  --phred33 --strand-specific ../1.clean.fq ../2.clean.fq\
              ./trinity_out_dir${i}/Trinity.fasta \
              hope-trinity_out_dir${i} 300
```
è¯„ä¼°ç»“æœè§£é‡Šè§ï¼š**<a href="http://deweylab.biostat.wisc.edu/detonate/rsem-eval.html" target="_blank">RSEM-EVAL: A novel reference-free transcriptome assembly evaluation measure</a>**ã€‚

<li>RSEM-EVAL produces the following three score related files: 'sample_name.score', 'sample_name.score.isoforms.results' and 'sample_name.score.genes.results'.</li>
<li>'sample_name.score' stores the evaluation score for the evaluated assembly. The first lines Scoreï¼šthe RSEM-EVAL score.</li>
<li>Higher RSEM-EVAL scores are better than lower scores. This is true despite the fact that the scores are always negative. For example, a score of -80000 is better than a score of -200000, since -80000 > -200000.</li>
##BUSCO explore completeness according to conserved ortholog
``` bash
git clone https://gitlab.com/ezlab/busco.git
```
ç‚¹å‡»**<a href="http://busco.ezlab.org/" target="_blank">BUSCOå®˜ç½‘</a>**ç›¸åº”å›¾æ ‡ä¸‹è½½æ‰€éœ€æ•°æ®åº“ã€‚
![](http://7xk19o.com1.z0.glb.clouddn.com/busco.png)
``` bash
python BUSCO.py -i SEQUENCE_FILE -o OUTPUT_NAME -l LINEAGE -m tran
```
SEQUENCE_FILEï¼š transcript set (DNA nucleotide sequences) file in FASTA format
OUTPUT_NAMEï¼š name to use for the run and temporary files (appended)
LINEAGEï¼š location of the BUSCO lineage data to use (e.g. fungi_odb9)
**å¯Ÿçœ‹ç»“æœ: åœ¨è¿è¡Œç»“æœæ–‡ä»¶å¤¹ä¸‹``short_summary_OUTPUT_NAME.txt``ä¸­æœ‰å¦‚ä¸‹ç»Ÿè®¡ä¿¡æ¯ğŸ‘‡**
```
C:80.0%[S:80.0%,D:0.0%],F:0.0%,M:20.0%,n:10

8 Complete BUSCOs (C)
8 Complete and single-copy BUSCOs (S)
0 Complete and duplicated BUSCOs (D)
0 Fragmented BUSCOs (F)
2 Missing BUSCOs (M)
10 Total BUSCO groups searched
```
ä¹Ÿå¯å›¾åƒåŒ–å±•ç¤ºç»“æœğŸ‘‡ï¼š
```
cp short_summary_OUTPUT_NAME.txt ./plot
python2.7 BUSCO_plot.py -wd ./busco/plot/
```
![](http://7xk19o.com1.z0.glb.clouddn.com/busco2.png)