title: Retroelements
date: 2016-01-03 20:18:31
tags: retroelements
categories: molecular biology
---
##Introduction
Retroelements are mobile genetic elements (MGEs) that retrotranspose via a RNA intermediate that is reverse-transcribed to DNA by the encoded reverse transcriptase and integrated into a new location within the host genome by an integrase enzyme. They have been found among different organisms from bacteria to humans and often constitute a significant part of genomes, particularly in higher plants and fungi. Various retroelements with different gene organizations and replicative mechanisms have evolved in the course of evolution. Although in most cases they have no effect on the host organism, there are many examples of mutations caused by retroelements resulting in various diseases. However co-adaptation has led in some cases to the use of retroelements for essential and beneficial host functions.

Retroelements (retrotransposons and retroviruses) can currently be divided into four systems or groups commonly known as the long terminal repeat (LTR) retroelements, the non-LTR retroelements, the tyrosine recombinase (YR) retroelements and the Penelope retrotransposons (Eickbush and Jamburuthugoda 2008).
##LTR retroelements
These include the broad range of LTR retrotransposons and retroviruses circulating in plants, fungi and animals. A full-lenght consensus LTR retroelement genome is characterized by an internal translating region (gag and pol genes, and env gene when is present) flanked by long terminal repeats (LTRs). They can be classified into four major groups or families based on sequence similarity and other features known as the Ty1/Copia, the Ty3/Gypsy, the Bel/Pao, and the Retroviridae families.
![](http://7xk19o.com1.z0.glb.clouddn.com/LTR_retroel.jpg)
###补充材料
![](http://7xk19o.com1.z0.glb.clouddn.com/env.png)
![](http://7xk19o.com1.z0.glb.clouddn.com/pictsimpleproteins.jpg)
Gag is a polyprotein and is an acronym for Group Antigens (ag).
Pol is the reverse transcriptase.
Env in the envelope protein.
The group antigens form the viral core structure, RNA genome binding proteins, and are the major proteins comprising the nucleoprotein core particle. Reverse transcriptase is the essential enzyme that carries out the reverse transcription process that take the RNA genome to a double-stranded DNA preintegrate form. The reverse transcriptase gene also encodes an Integrase activity and an RNase H activity that functions during genome reverse transscription.
##Non-LTR retroelements
<p>These constitute a system of retrotransposons widely distributed in eukaryotes, which do not present LTRs or terminal repeats; non-LTR retrotransposons end most frequently with a <i>poly(A)</i> tail at their 3′ end, while their 5′ end often contains variable deletions (5′ truncations). Depending on their capability to transpose or not autonomously these elements are classified as autonomous and non autonomous retroelements, respectively.
</p><p><br />
<b>Autonomous non-LTR retroelements</b>. On the basis of their molecular structures the autonomous non-LTR retroelements have been grouped in two major classes:
</p><p><u>R2 elements </u>: these constitute one of the most studied families of non-LTR retroelements (<a href="/index.php?title=Literature:100587&amp;action=edit&amp;redlink=1" class="new" title="Literature:100587 (page does not exist)">Eickbush 2002</a>). They encode for a single ORF with a central RT domain and an endonuclease (EN) conserved domain at the C-terminus.
</p><p><u>Long INterspersed repetitive Elements (LINEs) </u>: these are a family of 6-8 kb long elements encoding for two ORFs. The first ORF shows similarity to retroviral <i>gags</i>. The second ORF encodes for a pol polyprotein displaying RT and apurinic-apyrimidinic endonuclease (APE) domains. However, some lineages additionally include a protein domain of unknown function at the C-terminus and in other cases an RNase H (RH) downstream of the RT domain as usual in LTR retrotransposons (<a href="/index.php?title=Literature:88885&amp;action=edit&amp;redlink=1" class="new" title="Literature:88885 (page does not exist)">Eickbush and Jamburuthugoda 2008</a>). LINEs are widespread in mammals (<a href="/index.php?title=Literature:100588&amp;action=edit&amp;redlink=1" class="new" title="Literature:100588 (page does not exist)">Moran and Gilbert 2002</a>).  
</p>
![](http://7xk19o.com1.z0.glb.clouddn.com/Non-LTRs1.jpg)
<b>Non autonomous non-LTR retroelements</b>. These are DNA sequences of 80 to 630 bp known as Short INterspersed repetitive Elements (<b>SINEs</b>). They represent reverse-transcribed RNA molecules originally transcribed by RNA polymerase III into tRNA, rRNA, and other small nuclear RNAs (<a href="/index.php?title=Literature:46554&amp;action=edit&amp;redlink=1" class="new" title="Literature:46554 (page does not exist)">Malik and Eickbush 1998</a>). Like the LINEs, the SINEs end by a poly(A) tail or by A- or T-rich sequences, their 5' and 3' ends reveal similarities to tRNA genes (or, as shown for some animal SINEs, to 7SL RNA gene) and to the 3' end of LINEs, respectively (<a href="/index.php?title=Literature:100584&amp;action=edit&amp;redlink=1" class="new" title="Literature:100584 (page does not exist)">Oshima <i>et al</i>. 1996</a>). Two conserved sequence motifs found in the SINE tRNA-like part, called box A and box B, show homology to RNA polymerase III promoters. SINEs do not encode their own reverse transcriptase and are therefore unable to transpose autonomously. For this reason it has been proposed that SINEs use the enzymatic machinery of LINEs for their retrotransposition (<a href="/index.php?title=Literature:45266&amp;action=edit&amp;redlink=1" class="new" title="Literature:45266 (page does not exist)">Luan <i>et al.</i> 1993</a>; <a href="/index.php?title=Literature:99195&amp;action=edit&amp;redlink=1" class="new" title="Literature:99195 (page does not exist)">Wallace <i>et al.</i> 2008</a>; <a href="/index.php?title=Literature:92284&amp;action=edit&amp;redlink=1" class="new" title="Literature:92284 (page does not exist)">Kroutter <i>et al.</i> 2009</a>). LINEs and SINEs elements have been not only described in the genomes of animals but also in many species across the Plantae Kingdom (<a href="/index.php?title=Literature:67257&amp;action=edit&amp;redlink=1" class="new" title="Literature:67257 (page does not exist)">Schmidt 1999</a>).
</p>
![](http://7xk19o.com1.z0.glb.clouddn.com/Non-LTRs2.jpg)
##Tyrosine recombinase (YR) retroelements
<p>YR retroelements have been found in plants, protists, fungi, as well as a variety of animals including vertebrates, echinoderms and nematodes. The "gag-RT-RNaseH" genome organization of YR retroelements is similar to that of LTR retroelements but differ in the fact that YR retroelements lack the PR and usually show a <i>tyrosine recombinase</i> (instead of INT), which is typically involved in site-specific recombinations between similar or identical DNA sequences. YR retroelements can be divided into three families <b>DIRS</b>, <i><b>Ngaro</b></i> and <b>VIPER</b>  (<a href="/index.php?title=Literature:26665&amp;action=edit&amp;redlink=1" class="new" title="Literature:26665 (page does not exist)">Goodwin <i>et al</i>. 2004</a>; <a href="/index.php?title=Literature:26666&amp;action=edit&amp;redlink=1" class="new" title="Literature:26666 (page does not exist)">Goodwin and Poulter 2004</a>; <a href="/index.php?title=Literature:79660&amp;action=edit&amp;redlink=1" class="new" title="Literature:79660 (page does not exist)">Vazquez <i>et al</i>. 2000</a>; <a href="/index.php?title=Literature:44948&amp;action=edit&amp;redlink=1" class="new" title="Literature:44948 (page does not exist)">Lorenzi <i>et al</i>. 2006</a>). The typical structure of DIRS elements contains inverted terminal repeats (ITRs), internal ORFs and an internal complementary region (ICR) derived from the duplication of flanking ITR sequences. DIRS elements differ from the two other YR-like families in the presence of a conserved methyltransferase (MT) domain at C-terminal to RT/RH that is similar to those MTs encoded by various bacteriophages (<a href="/index.php?title=Literature:26666&amp;action=edit&amp;redlink=1" class="new" title="Literature:26666 (page does not exist)">Goodwin and Poulter 2004</a>). <i>Ngaro</i> elements show an ORF organization similar to that of DIRS elements but differ in the orientation of flanking repeats. While DIRS elements are delimited by inverted repeats, <i>Ngaro</i> elements are flanked by direct repeats (referred as A1, A2, B1 and B2). <i>Ngaro</i> elements have been found in different organisms as the zebrafish <i>Danio rerio</i> (Dr<i>Ngaro</i>1), as well as in fungi and in echinoderms (<a href="/index.php?title=Literature:26666&amp;action=edit&amp;redlink=1" class="new" title="Literature:26666 (page does not exist)">Goodwin and Poulter 2004</a>). In turn, VIPER (Vestigial interposed retroelement) elements have been described in the genomes of trypanosome protozoan parasites. As an example, the figure below also shows the genomic organization of <i>tcVIPER</i>, an element described in the genome of <i>T. cruzi</i>. This element contains a coding internal region flanked by a lineage of SINEs (called SIRE) also found in <i>T. cruzi</i> genome (<a href="/index.php?title=Literature:44948&amp;action=edit&amp;redlink=1" class="new" title="Literature:44948 (page does not exist)">Lorenzi <i>et al</i>. 2006</a>; <a href="/index.php?title=Literature:79660&amp;action=edit&amp;redlink=1" class="new" title="Literature:79660 (page does not exist)">Vazquez <i>et al</i>. 2000</a>).
![](http://7xk19o.com1.z0.glb.clouddn.com/YR_retroel.jpg)
##Penelope retrotransposons (PLEs)
<p>This is a family of retrotransposons described in many animal genomes (more than 80 species belonging to at least 10 animal phyla), protists, fungi, and plants (<a href="/index.php?title=Literature:100585&amp;action=edit&amp;redlink=1" class="new" title="Literature:100585 (page does not exist)">Arkhipova 2006</a>). Their genome structure contains apparent LTRs that may be in either direct or inverted orientations flanking a coding region with RT and EN domains (i.e. a <i>pol</i> polyprotein domain). Phylogenetic reconstruction analysis indicate that PLEs-like RTs are closer to telomerase RTs (TERTs) than to any other characterized RTs (<a href="/index.php?title=Literature:2404&amp;action=edit&amp;redlink=1" class="new" title="Literature:2404 (page does not exist)">Arkhipova <i>et al.</i> 2003</a>). In turn, PLE-like ENs are related with the intron-encoded endonucleases and the bacterial repair endonuclease UvrC, both belonging to the <i>Uri</i> family of ENs (<a href="/index.php?title=Literature:61392&amp;action=edit&amp;redlink=1" class="new" title="Literature:61392 (page does not exist)">Pyatkov <i>et al.</i> 2004</a>). Studies realized with the fly <i>Drosophila virilis</i> and bdelloid rotifer organisms revealed that the majority of PLEs in these species contain spliceosomal introns (<a href="/index.php?title=Literature:2404&amp;action=edit&amp;redlink=1" class="new" title="Literature:2404 (page does not exist)">Arkhipova <i>et al.</i> 2003</a>). The peculiar structural organization of PLE elements, their ability to retain introns during transposition, and the distinct placement in the phylogeny of retroelements, suggest that PLEs constitute an ancient class of retroelements (<a href="/index.php?title=Literature:100585&amp;action=edit&amp;redlink=1" class="new" title="Literature:100585 (page does not exist)">Arkhipova 2006</a>; <a href="/index.php?title=Literature:100586&amp;action=edit&amp;redlink=1" class="new" title="Literature:100586 (page does not exist)">Schostak <i>et al.</i> 2008</a>).
![](http://7xk19o.com1.z0.glb.clouddn.com/Penelope_retroel.jpg)
![](http://7xk19o.com1.z0.glb.clouddn.com/Legend.jpg)

Read more: http://gydb.org/index.php/Retroelements